packages/ckeditor5-image/src/imageupload/utils.js

```diff
@@ -27,28 +29,22 @@ export function createImageTypeRegExp( types ) {
 }
 
 /**
- * Creates a promise that fetches the image local source (Base64 or blob) and resolves with a `File` object.
+ * Creates a promise that converts the image local source (Base64 or blob) to a blob and resolves with a `File` object.
  *
- * @param {module:engine/view/element~Element} image Image whose source to fetch.
- * @returns {Promise.<File>} A promise which resolves when an image source is fetched and converted to a `File` instance.
+ * @param {module:engine/view/element~Element} image Image whose source to convert.
+ * @returns {Promise.<File>} A promise which resolves when an image source is converted to a `File` instance.
  * It resolves with a `File` object. If there were any errors during file processing, the promise will be rejected.
  */
-export function fetchLocalImage( image ) {
-	return new Promise( ( resolve, reject ) => {
-		const imageSrc = image.getAttribute( 'src' );
-
-		// Fetch works asynchronously and so does not block browser UI when processing data.
-		fetch( imageSrc )
-			.then( resource => resource.blob() )
-			.then( blob => {
-				const mimeType = getImageMimeType( blob, imageSrc );
-				const ext = mimeType.replace( 'image/', '' );
-				const filename = `image.${ ext }`;
-				const file = new File( [ blob ], filename, { type: mimeType } );
-
-				resolve( file );
-			} )
-			.catch( reject );
+export function createImageFile( image ) {
```

Comment:
The minor breaking change has been added to suggested commit message.
IMHO we should change the name, because in this PR we are no longer fetching an image, but we are converting it locally.